roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -94,12 +112,52 @@
     annotations:
       service.alpha.openshift.io/serving-cert-secret-name: alertmanager-tls
     ports:
-      - name: prometheus
-        port: "{{ openshift_prometheus_service_port }}"
-        targetPort: "{{ openshift_prometheus_alertmanager_service_targetport }}"
-        protocol: TCP
+      - port: 443
+        targetPort: 9443
+  with_items:
+    - name: alerts
+
+- name: Create prometheus-node-exporter service
```

Comment:
Please follow the template model as described for openshift-web-console.  Template, minimal parameters, set of yedit patches for anything that is necessary to be parameterized.  Most things shouldn't be parametrizable.

Reply:
The template should work ootb on Origin (so it should be an effective copy).  Policy and RBAC can be split.

@sdodson do we have any sort of rudimentary write up of what the component model looks like?

Reply:
No, not yet. There's a proposal drafted that moves straight on to APBs but I don't think it's consumable yet.
We need to write this up sooner rather than later though as I don't think we're going to switch to APBs before we add more components.